src/reporting/feedback.tsx

```diff
@@ -142,37 +142,21 @@ const Feedback: FC = () => {
 	const [t] = useTranslation();
 	const topics = useMemo(() => getTopics(t), [t]);
 	const allApps = useAppList();
-	const [feedbackPermission, setFeedbackPermission] = useState(false);
+	const feedbackPermission = useAllConfigStore(
+		(state) =>
+			state.getConfigByKey('carbonioSendFullErrorStack') === 'TRUE' &&
+			state.getConfigByKey('carbonioSendAnalytics') === 'TRUE' &&
+			state.getConfigByKey('carbonioAllowFeedback') === 'TRUE'
```

Comment:
@Tarunmeel94 you can use TRUE variable from constant file instead of write static keyword here.